src/functions/symbolsOp.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import {defineFunctionBuilders} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+
+import * as mml from "../buildMathML";
+
+import type Options from "../Options";
+import type ParseNode from "../ParseNode";
+import type {Group} from "../symbols";
+
+// Operator ParseNodes created in Parser.js from symbol Groups in src/symbols.js.
+
+function defineOpFun<NODETYPE: Group>(
```

Comment:
What does `fun` mean?

Reply:
Function. Renamed to `defineOpFunction` for clarity.